Form.css

```diff
@@ -0,0 +1,24 @@
+.Form {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.Form * {
```

Comment:
Using the universal selector (`*`) can cause performance issues. But, more importantly, it has the potential to cause unintended side-effects by applying styles to elements that weren't intended or haven't yet been added to the view.